projects/api-client/src/model/mainBuild.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Wharf main API
+ * Wharf backend API that manages data storage for projects, builds, providers, etc.
+ *
+ * OpenAPI spec version: v4.2.0
+ * Contact: wharf@iver.se
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+import { MainBuildParam } from './mainBuildParam';
```

Comment:
There are still some stray changes here to the regenerated types. Maybe they're the remenants of something like d123234d9bdabb3019d740a84edeba3b33852368 or 18396604b95bb4b94845385029c1a6e68da994f6.

Doing a short `git diff origin/master --name-only | grep '^projects/'` from your branch reveals that it's only these files:

```
projects/api-client/src/model/mainBuild.ts
projects/import-azuredevops-client/src/model/responseBuildReferenceWrapper.ts
projects/import-github-client/src/model/problemResponse.ts
```

Could you remove those 3 files?